<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="GLFrame" name="glframe" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuMouse">
    <property name="title">
     <string>Mouse</string>
    </property>
    <addaction name="actionCamera_FPS"/>
    <addaction name="actionCamera_Orbit"/>
    <addaction name="separator"/>
    <addaction name="actionForce"/>
    <addaction name="actionTorque"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMouse"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCamera_FPS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Camera FPS</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionCamera_Orbit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Camera Orbit</string>
   </property>
   <property name="shortcut">
    <string>V</string>
   </property>
  </action>
  <action name="actionForce">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Force</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionTorque">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Torque</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLFrame</class>
   <extends>QWidget</extends>
   <header location="global">interface/glframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
